.ci/jenkins_job_build.sh

```diff
@@ -85,6 +91,15 @@ fi
 # - Call checkcommits.
 bash "${GOPATH}/src/${cc_repo}/.ci/setup.sh"
 
+if [ -n "$pr_number" ]
+then
+	# Now that checkcommits has run, merge the PR into the master branch
+	# before running the tests. This is required to ensure coveralls works.
+	git checkout master
+	git merge --ff-only -m "CI merge of PR $pr_number" "$pr_branch"
+	git branch -d "$pr_branch"
```

Comment:
if you want to be paranoid, then maybe `-D` instead of `-d` (adds a `--force`)

Reply:
I guess we could go with `-D`, but I was thinking the current approach was paranoid since this will fail if the previous merge failed (which it can't do due to `set -e`, but... paranoia ;)

Reply:
Why do a merge when you could do a simple reset ?
```bash
git checkout master
git reset --hard "$pr_branch"
git reset -D "$pr_branch"
```

Reply:
@jodh-intel what about this comment ?